app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,8 @@
   <string name="admin_auth_sub">Enter the Administrator PIN in order to create a new account.</string>
```

Comment:
Please update admin_auth_sub string according to mock.

Reply:
oh, my bad, Updated.